packages/bcp/src/address.spec.ts

```diff
@@ -17,23 +17,6 @@ describe("address", () => {
     });
   });
 
-  describe("isValidAddress", () => {
```

Comment:
Yeah, thanks for cleaning this up, there were some loose ends to simplify when I pulled out this logic